libstatistics_collector/test/topic_statistics_collector/test_received_message_age.cpp

```diff
@@ -52,22 +52,24 @@ std_msgs::msg::String GetStringMessageWithoutHeader()
 }
 }  // namespace
 
+using ReceivedImuMessageAgeCollector = libstatistics_collector::
+  topic_statistics_collector::ReceivedMessageAgeCollector<sensor_msgs::msg::Imu>;
+using ReceivedStringMessageAgeCollector = libstatistics_collector::
+  topic_statistics_collector::ReceivedMessageAgeCollector<std_msgs::msg::String>;
```

Comment:
Suggestion: move this into unnamed namespace

Reply:
Done.